engines/python/setup/djl_python/tests/test_properties_manager.py

```diff
@@ -101,6 +101,38 @@ def test_tnx_configs(self):
         self.assertEqual(tnx_configs.compiled_graph_path,
                          str(properties['compiled_graph_path']))
 
+    def test_tnx_all_configs(self):
+        # TODO: Replace with actual example of context_length_estimate
+
+        properties = {
+            "n_positions":
+            "2048",
+            "load_split_model":
+            "true",
+            "load_in_8bit":
+            "true",
+            "compiled_graph_path":
+            "s3://test/bucket/folder",
+            "low_cpu_mem_usage":
+            "true",
+            'context_length_estimate':
+            '{"context_length": "128", "variable_size": "12"}'
```

Comment:
@tosterberg Added temp context_length_estimate for testing whether we are parsing the properties.